### ISSUE TITLE
Run android "cannot find symbol"

### ISSUE THREAD

**Issue by iamn1ck** [WrathChaos/react-native-typescript-boilerplate#3]
I just setup andoid-studio and the android sdk on my machine. And when running `yarn android` get this error

```
 error: cannot find symbol
        protected JSIModulePackage getJSIModulePackage() {
                  ^
  symbol: class JSIModulePackage
```

Any idea what is wrong?

**Comment by kuraydev** [comment]
Hello @iamn1ck, 
I need to check it out, thank you for reporting

**Comment by kuraydev** [comment]
Hello again @iamn1ck,

I released a new version 2.0.2. 
Please check it out [Release 2.0.2](https://github.com/WrathChaos/react-native-typescript-boilerplate/releases/tag/2.0.2)

I fixed many Android issues, it works great now :) 
You should set your own SDK path on Android's `local.properties` file.
